.github/workflows/integration-gpu.yaml

```diff
@@ -0,0 +1,118 @@
+name: GPU integration CI pipeline
+
+on:
+  pull_request:
+
+jobs:
+  build-rock:
+    uses: ./.github/workflows/build_gpu.yaml
+
+  start-runner:
+    name: Start self-hosted EC2 runner
+    needs: [build-rock]
+    runs-on: ubuntu-22.04
+    outputs:
+      label: ${{ steps.start-ec2-runner.outputs.label }}
+      ec2-instance-id: ${{ steps.start-ec2-runner.outputs.ec2-instance-id }}
+    steps:
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v4.0.2
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ secrets.AWS_REGION }}
+      - name: Start EC2 runner
+        id: start-ec2-runner
+        uses: machulav/ec2-github-runner@v2.3.6
+        with:
+          mode: start
+          github-token: ${{ secrets.WELPAOLO_GITHUB_TOKEN }}
```

Comment:
I believe this is a temporary name for the secret, right?

Reply:
We keep this token just for a limited amount of time. We need to figure out some details on how to handle it properly with  the dataplatform account